pkg/plugin/interfaces.go

```diff
@@ -105,3 +105,13 @@ type CreateWebhookPluginGetter interface {
 type CreateWebhook interface {
 	GenericSubcommand
 }
+
```

Comment:
```suggestion
// EditPluginGetter is an interface that defines gets an Edit plugin
```

Reply:
The rest of the `XXXXPluginGetter` don't have these comments. IMHO, either place them in all of them or leave them without it.

Reply:
All exported methods required to be doc/commented, otherwise, it will not pass in the lint. See that all getters in this file are properly commented.

Reply:
I don't know why I thought the rest didn't have it. I was the one suggesting him to remove these two comments. Sorry @prafull01 but you will need to add them again. Blame me for that.

Reply:
Hi @Adirio, 

It is fine. Really thank you for your help in the review 👍

Reply:
It has happened because my branch was quite behind from master. @Adirio So not your mistake. Thanks for the review :)

Reply:
Fixed it